controllers/core/openstackcontrolplane_controller.go

```diff
@@ -224,6 +226,13 @@ func (r *OpenStackControlPlaneReconciler) reconcileNormal(ctx context.Context, i
 		return ctrlResult, nil
 	}
 
+	ctrlResult, err = openstack.ReconcileDataplaneNode(ctx, instance, helper)
```

Comment:
Why are we reconciling Dataplane nodes as part of the controlplane? I thought these were meant to be managed as part of the DataPlane CR?

Reply:
oh, my bad! I was following the pattern of the other operators and didn't realize it was the controlplane